### DISCUSSION TITLE
markupsafe compatibility issue with Airflow 2.3.3

### DISCUSSION THREAD

**Discussion by akashgangulyhf** [astronomer/astronomer-providers#528] [Q&A] [answered]
Hello, currently the markupsafe supports (<2.1.0,>=1.1.1) but as per Airflow 2.3.3 [dependency constraints](https://raw.githubusercontent.com/apache/airflow/constraints-2.3.3/constraints-3.10.txt), we need MarkupSafe==2.1.1. Can you please bump up the version check?

**Comment by phanikumv** [comment]
Hi @akashgangulyhf , 
Thanks for starting the discussion !
We will look into this and bump up the markupsafe version.

**Reply by akashgangulyhf** [reply]
Thanks @phanikumv for taking this up. Please bump up other libs as well if there are any. I only mentioned markupsafe as I faced issue with that but it tends to come sequentially. More might be there.

**Reply by kaxil** [reply]
We don't have any other libraries that are pinned as shown below :) :

https://github.com/astronomer/astronomer-providers/blob/85d2941fb41f1dbbc6c00eb122a9de9e902ec895/setup.cfg#L36-L127

**Comment by phanikumv** [comment, marked as answer]
Hi @akashgangulyhf we have released astronomer providers 1.7.1 which should fix this issue.
